tests/files/packet_debian12-cilium.yml

```diff
@@ -5,3 +5,11 @@ mode: default
 
 # Kubespray settings
 kube_network_plugin: cilium
+
+kube_vip_enabled: true
+kube_vip_controlplane_enabled: true
+kube_vip_address: 10.20.30.40  # FIXME
```

Comment:
This has to be changed. What is the subnet of the VMs?

Reply:
Hmm IIRC it's based on the sample inventory if you could have a template that pick a reasonable address index within `kube_pods_subnet` or `kube_service_addresses` that would be nice I think

Reply:
AH wait nvm the subnet of the vms hmmm in that case I am not sure, afaik kubevirt decides not sure if you can get an IP at that layer like that :/